incubator/astro/templates/configmap.yaml

```diff
@@ -0,0 +1,19 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ include "astro.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "astro.name" . }}
+    helm.sh/chart: {{ include "astro.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+data:
+  DD_API_KEY: {{ .Values.datadog.apiKey }}
+  DD_APP_KEY: {{ .Values.datadog.appKey }}
```

Comment:
It looks like we've got these keys both in the configmap and the secret - is that intended?

Reply:
no, that should not be in the configmap too.  I have removed it and it's now only in the secret.  Nice find!